study.md

```diff
@@ -24,18 +24,19 @@ After reading all of the required readings, please tell us anything that you
 were confused about.
 
 ```md
-<!-- your answer here -->
+1)I was confused on anonymous functions. I get that function expressions can be used to store that in a variable and that there is no actual name for the function, but I'm not sure how that would be beneficial in application.
+2)I was confused on how return true and return false works and when we would use them.
 ```
 
 Are there any nagging questions related to functions that you have?
 
 ```md
-<!-- your answer here -->
+Not particularly, I need to see a real life application and visuals to get a better feel for how functions work
 ```
 
 Thus far, how are you feeling about your progress as a developer? Please give an
 extensive answer.
 
 ```md
-<!-- your answer here -->
+I was feeling very confident until functions were introduced. At a VERY basic level, understanding how different JS syntax works alone is easy to comprehend. However, incorporating those within each other(such as using nesting functions within arrays, using loops and conditionals within a function and even wriiting out the syntax for complex functions/conditionals/loops etc) is confusing and I freeze up when given a certain time to finish it by(such as the diagnostics). I feel as though I'm not progressing as quickly as the others and a bit concerned that I'll fall behind as the weeks go by.
 ```
```

Comment:
If you feel like you are struggling then please contact a Mike to set up a 1 on 1 with either himself or a DIR. Do not let yourself struggle silently. You got this! 🦄